src/components/flowchart/draw.js

```diff
@@ -180,8 +180,12 @@ export const drawNodes = function(changed) {
       .duration(this.DURATION)
       .attr('transform', node => `translate(${node.x}, ${node.y})`)
       .on('end', () => {
-        // Sort nodes so tab focus order follows X/Y position
-        allNodes.sort((a, b) => a.order - b.order);
+        try {
+          // Sort nodes so tab focus order follows X/Y position
+          allNodes.sort((a, b) => a.order - b.order);
+        } catch (ex) {
```

Comment:
`ex`? Haven't seen that before, would've thought `err` usually, but whatever it really doesn't matter

Reply:
I just tend to write `ex` for exceptions that must be handled, `err` for anything else to be clear. Anyway, I changed it to `err` then.

Reply:
aha. ok no worries, thanks!